scripts/test.sh

```diff
@@ -31,6 +31,7 @@ run () {
     yarn mocha $(find src/mobile/components/**/*/test -name '*.coffee')
     yarn mocha $(find src/mobile/apps/*/test -name '*.coffee')
     yarn mocha $(find src/mobile/apps/*/**/*/test -name '*.coffee')
+    yarn jest $(find src/desktop/components -name '*.jest.js')
```

Comment:
We should scope this to the full `/desktop` app:
```
yarn jest $(find src/desktop -name '*.jest.js')
```

Reply:
Actually, something I just remembered from Positron is that directory scanning is unnecessary as it's [built into jest](https://github.com/artsy/positron/blob/master/scripts/test.sh#L9). Need to make sure that the `roots` property pushes in `src` in [alongside](https://facebook.github.io/jest/docs/en/configuration.html#roots-array-string) the default tho.

Reply:
Yeah makes sense -- moved the regex into `package.json`, so we can run anything from `/src` with `.jest.` using `yarn test`